src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -545,8 +574,15 @@ public boolean isDebugGraphTransformations() {
 
     public boolean isRecoverDanglingBranches() { return recoverDanglingBranches; }
 
-    public void setDebugGraphTransformations(final boolean debugGraphTransformations) {
-        this.debugGraphTransformations = debugGraphTransformations;
+    public void setDebugHistogramOutput(final File file) {
+        this.debugHaplotypeFinding = true;
+        this.graphHaplotypeHistogramPath = file;
+        this.haplotypeHistogram = new Histogram(1.0);
+        this.kmersUsedHistogram = new Histogram(1.0);
+    }
+
+    public void setDebugGraphTransformations(final boolean debugHaplotypeFinding) {
```

Comment:
This method is no longer used and not needed. Also, you can get rid of the `debugHaplotypeFinding` boolean field in this class and just use `graphHaplotypeHistogramPath` to detect whether debugging output is on.

Reply:
done